docs/dev grade services/scripts/mongo

```diff
@@ -21,9 +21,9 @@ docker run --name ${MONGO_CONTAINER_NAME} ${MONGO_DATA_CMD} -d ${MONGO_VERSION}
     CONTAINER_ID=`docker ps|grep -w ${MONGO_CONTAINER_NAME}|awk '{print $1}'`
     echo "container id: ${CONTAINER_ID}"
 
-    docker network connect hcf ${CONTAINER_ID}
+    docker network connect scf ${CONTAINER_ID}
```

Comment:
I would be very surprised if this worked

Reply:
Yes, well, I do not know what I am doing here, and it is for something we do not use AFAIK, so I did not think too much on the change.
I suspect it needs setup from other (unknown to me) parts of the system.